CMakeLists.txt

```diff
@@ -40,6 +40,61 @@ else()
   message(WARNING "Unknown compiler... recklessly proceeding without a version check")
 endif()
 
+set(LIBMONGOC_REQUIRED_VERSION 1.22.1) # TODO: update to 1.24.0 once released.
+set(LIBMONGOC_REQUIRED_ABI_VERSION 1.0)
+
+set(NEED_DOWNLOAD_C_DRIVER false CACHE INTERNAL "")
```

Comment:
Use a regular variable instead of `CACHE`, since this is unconditionally assigned.

Reply:
Ok, this seems to be resolved with this commit: https://github.com/mongodb/mongo-cxx-driver/pull/967/commits/fec94f4ae58c790d2edc67245b55d6f2f429a811